### ISSUE TITLE
Failed to save result: possible bot detected

### ISSUE THREAD

**Issue by davebro** [monkeytypegame/monkeytype#4584]
### Did you clear cache before opening an issue?

- [X] I have cleared my cache

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Does the issue happen when logged in?

Yes

### Does the issue happen when logged out?

Yes

### Does the issue happen in incognito mode when logged in?

Yes

### Does the issue happen in incognito mode when logged out?

Yes

### Account name

dbro129

### Current Behavior

My results are not being saved because it thinks I'm a bot. Am I really that good? My speed is generally somewhere between 140-150+ wpm. I'll get halfway through a 60 sec test and my cursor will freeze. I'll keep typing and once I get to the end of the second line, it unfreezes. But once the test finishes, it displays an error:

`Failed to save result: possible bot detected.`

Do I need to do something to _not_ be detected as a bot?

### Environment

- OS: MacOS Ventura 13.4.1
- Browser: Chrome Version 116.0.5845.96 (Official Build) (x86_64)
- Browser Version:

**Comment by Miodec** [comment]
Please send an example result which triggers this error

**Comment by lukew3** [comment]
I've had the issue where the cursor stops moving too. Usually it happens during my first test on and maybe 10 words in. It probably has to do with the main thread being blocked by something. If the anticheat stops working while the thread is blocked, the anticheat will see many characters appear at the same time, and will trigger a bot detection.

**Comment by Miodec** [comment]
Only on the first test?

**Comment by lukew3** [comment]
> Only on the first test? 

I can't remember exactly but I think so. Also, I was using Firefox when I had this error and had a lot of extensions like ublock which could also be part of the issue.

**Comment by kspcode** [comment]
Same thing happened but later I found it was saved.

![Annotation 2023-08-30 172257](https://github.com/monkeytypegame/monkeytype/assets/143815103/72a011ea-781f-46f3-8f53-81ec517670d0)


On 30th it did not save.
But today it shows that it saved.

![Annotation 2023-09-01 172001](https://github.com/monkeytypegame/monkeytype/assets/143815103/c7dc22af-8856-4144-a199-ee825fd483af)

**Comment by Miodec** [comment]
Closing due to inactivity.

@lukew3 if you run into your issue again open a new issue please.

**Comment by Nusab19** [comment]
Happened always in `15s` tests, (with 90 to 100WPM).

So I did a `120s` test, pretty slowly, with mistakes, to not get much high WPM. But still, got the error. Started happening just today!

OS: `Manjaro KDE`
Browser: `1.89.145 Chromium: 147.0.7727.137 (Official Build) (64-bit)`

For me, doesn't happen when logged out. -.-

<img width="1920" height="942" alt="Image" src="https://github.com/user-attachments/assets/e1a66681-9e44-49cf-862a-1c8c7b86a121" />

**Comment by natescode** [comment]
This happened to me today for the first time. cleared cache, incognito etc. It only happens when I'm logged in. I'm on Linux, Firefox 152.0 and use Dvorak. I've typed dozens of hours with this configuration. I'm auto banned now apparently. I'd join discord but cannot verify my phone number because rate limiting; I'll try later. 

<img width="1088" height="1388" alt="Image" src="https://github.com/user-attachments/assets/6d165572-bd99-45d8-ae8f-b8d4360c444f" />
